cartography/intel/gcp/dns.py

```diff
@@ -121,6 +114,134 @@ def get_dns_rrs(dns: Resource, dns_zones: List[Dict], project_id: str) -> List[R
             # raise
             return []
 
+@timeit
+def transform_rrs(rrsets: list, zone: dict, project_id: str):
```

Comment:
`list` or `dict` are invalid. Are the tests working properly? I will guess that Tests are failing. Please ensure all the tests are working properly.